vMenuServer/MainServer.cs

```diff
@@ -234,6 +236,19 @@ public MainServer()
         }
         #endregion
 
+        private void LogToDiscord([FromSource] Player player, string title, string data)
+        {
+            // Implement the log to Discord code
+            string desc = data;
+            string footer = "[" + player.Handle + "] " + player.Name;
+            string webhook = GetConvar("vMenu_Webhook", null);
```

Comment:
The convar doesn't match the naming scheme used for other convars.

Reply:
It's also not using the ConfigManager class to get the convar, which should be avoided.